tests/test-rest-comments-controller.php

```diff
@@ -1288,6 +1288,11 @@ protected function check_comment_data( $data, $context ) {
 		$this->assertEquals( mysql_to_rfc3339( $comment->comment_date_gmt ), $data['date_gmt'] );
 		$this->assertEquals( get_comment_link( $comment ), $data['link'] );
 		$this->assertContains( 'author_avatar_urls', $data );
+		$this->assertEqualSets( array(
+			'self',
+			'collection',
+			'up',
+		), array_keys( $links ) );
```

Comment:
I'd rather get these merged, as this PR is only adding tests, these are better than none